apps/src/templates/VersionRow.jsx

```diff
@@ -22,57 +26,72 @@ export default class VersionRow extends React.Component {
   }
 
   render() {
-    let button;
+    let buttons = [];
     if (this.props.isLatest) {
-      button = (
+      // this is the placeholder for the latest version (can't be resolved)
+      buttons.push(
         <button
+          key={buttons.length}
```

Comment:
It's better to use unique identifiers than indexes as keys (https://reactjs.org/docs/lists-and-keys.html#keys). An example of that here would be "latest-version-button"

Reply:
Oh man I didn't even notice this- thanks!